backend/engine/utils/plugin.py

```diff
@@ -475,7 +488,10 @@ def get_secret_raw_wl(scan):
 
 
 def get_secret_al(scan):
```

Comment:
could we add a `Scan` type here?

Reply:
Not yet, sadly -- this function uses a Django implicit attribute (`allowlistitem_set`) that can't be type-checked.  I added a comment so we can revisit this later:
```python
# Note: scan type is unspecified until we enable typechecking Django models.
```